bin/simulate_psf

```diff
@@ -1280,7 +1308,10 @@ def iteration_loop( pars, obi_info, src_info ):
         file name is stored in a psf.lis file that will get cleaned up
         at the end.
     """
-    for nn in range( int( pars["numiter"] )):
+
+    nn = 0
+    numrays = 0
+    while nn < int(pars["numiter"]) and nn < 10000:
```

Comment:
I would suggest making 10000 a variable (perhaps an optional argument to the routine) or  a constant, but IO'm not sure it's worth it!